ansible/redpanda/cluster/README.md

```diff
@@ -0,0 +1,193 @@
+# Ansible Collection for Redpanda
+
+Redpanda Ansible Collection that enables provisioning and managing a [Redpanda](https://www.redpanda.com/) cluster.
+
+## Usage
+
+### Required Steps: Deploying Redpanda
+
+> Note: This playbook is designed to be run repeatedly against a running cluster however be aware that it will cause a
+> rolling restart of the cluster to occur. If you do not want the cluster to restart you can
+> specify `restart_node=false`
+> either globally (as --extra-vars, or on a host-by-host basis in the `hosts.ini` file). If you make changes to the node
+> config and do not perform a restart then you may leave your `rpk` config (and this may inhibit subsequent executions
+> of
+> the playbook). If you re-run the playbook it will overwrite any configurations with those specified in the playbook,
+> so
+> care should be taken to ensure that the playbook contains any desired configuration (for example, if you have enabled
+> TLS on your cluster, any subsequent runs of `provision-node` should be made with `-e tls=true` otherwise the playbook
+> will disable TLS).
+
+Before running these steps, verify that the `hosts.ini` file contains the correct information for your infrastructure.
+This will be automatically populated if using the terraform steps above.
+
+1. `ansible-playbook --private-key <your_private_key> -i hosts.ini -v ansible/provision-node.yml`
+
+Available Ansible variables:
+
+You can pass the following variables as `-e var=value`:
+
+| Property                     | Default value                      | Description                                                                                                                                                                                                                                                                                               |
+|------------------------------|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `redpanda_organization`      | redpanda-test                      | Set this to identify your organization in the asset management system.                                                                                                                                                                                                                                    |
+| `redpanda_cluster_id`        | redpanda                           | Helps identify the cluster.                                                                                                                                                                                                                                                                               |
```

Comment:
[heads-up] https://github.com/redpanda-data/redpanda/pull/9713 will remove organization and cluster_id in the next redpanda release (v23.2) so we might as well remove it from here.

Reply:
I'll remove it in a few versions when it ages out I think